docker/llm/serving/cpu/docker/README.md

```diff
@@ -29,7 +28,74 @@ sudo docker run -itd \
         $DOCKER_IMAGE
 ```
 
-
 After the container is booted, you could get into the container through `docker exec`.
 
 To run model-serving using `BigDL-LLM` as backend, you can refer to this [document](https://github.com/intel-analytics/BigDL/tree/main/python/llm/src/bigdl/llm/serving).
+Also you can set environment variables and start arguments while running a container to get serving started initially. You may need to boot serveral containers to support. 
+
+To start a controller container:
+```bash
+#/bin/bash
+export DOCKER_IMAGE=intelanalytics/bigdl-llm-serving-cpu:2.5.0-SNAPSHOT
+controller_host=localhost
+controller_port=23000
+api_host=localhost
+api_port=8000
+sudo docker run -itd \
+        --net=host \
+	    --privileged \
+        --cpuset-cpus="0-47" \
+        --cpuset-mems="0" \
+        --memory="64G" \
+        --name=serving-cpu-controller \
+        --shm-size="16g" \
+	    -e ENABLE_PERF_OUTPUT="true" \
+        -e CONTROLLER_HOST=$controller_host \
+        -e CONTROLLER_PORT=$controller_port \
+        -e API_HOST=$api_host \
+        -e API_PORT=$api_port \
+        $DOCKER_IMAGE -m controller
+```
+To start a worker container:
+```bash
+#/bin/bash
+export DOCKER_IMAGE=intelanalytics/bigdl-llm-serving-cpu:2.5.0-SNAPSHOT
+export MODEL_PATH=YOUR_MODEL_PATH
+controller_host=localhost
+controller_port=23000
+worker_host=localhost
+worker_port=23001
+sudo docker run -itd \
+        --net=host \
+	    --privileged \
+        --cpuset-cpus="0-47" \
+        --cpuset-mems="0" \
+        --memory="64G" \
+        --name="serving-cpu-worker" \
+        --shm-size="16g" \
+	    -e ENABLE_PERF_OUTPUT="true" \
+        -e CONTROLLER_HOST=$controller_host \
+        -e CONTROLLER_PORT=$controller_port \
+        -e WORKER_HOST=$worker_host \
+        -e WORKER_PORT=$worker_port \
+        -e OMP_NUM_THREADS=48 \
+        -e MODEL_PATH=/llm/models/Llama-2-7b-chat-hf \
+	    -v $MODEL_PATH:/llm/models/ \
+        $DOCKER_IMAGE -m worker -w vllm_worker # use -w model_worker if vllm worker is not needed
+```
+
+Then you can use `curl` for testing, an example could be:
+```bash
+curl -X POST -H "Content-Type: application/json" -d '{
+    "model": "Llama-2-7b-chat-hf",
```

Comment:
Change this to "YOUR_MODEL_FULL_PATH"

Reply:
FastChat use the model dir name instead of full path?